build.gradle.kts

```diff
@@ -47,7 +47,7 @@ dependencies {
 }
 
 java {
-  toolchain.languageVersion.set(JavaLanguageVersion.of(16))
+  toolchain.languageVersion.set(JavaLanguageVersion.of(17))
```

Comment:
I got an error when setting this to `17` and kotlin to `16` so ended up setting language version to 16, but isn't related to the app insights error